CoreGraphFramework/src/au/gov/asd/tac/constellation/graph/undo/access/AttributeValueUpdater3.java

```diff
@@ -92,10 +92,12 @@ public void getUndo(UndoGraphEditState edit) {
         new ValueGetter() {
             @Override
             public void getExecute(UndoGraphEditState edit) {
+                // This method has been intentionally left blank
```

Comment:
This comment is inconsistent with other similar comments which read
```
// Default case when the position of the value getter is equal to 0
// It has been intentionally left blank
```

Reply:
@cygnus-x-1 I could be wrong but the valueGetters in all of the classes except for the two you've pointed out do nothing in index 0 so it appears that that's a default case to intentionally do nothing. For some reason in this class and in class IdValueUpdater3, the empty valueGetter is at position 2 of the array instead of 0 so its not the default case. I could not figure out why they've done it like this here. I can change the comment but I'm not sure what to change it to since I can't explain why they've done it differently. Let me know if I'm misunderstanding something.

Reply:
I missed that context from the snippets I was looking at. I also don't fully understand how the undo stack is working... so i'll take your best judgement here.